cppcache/include/geode/DistributedSystem.hpp

```diff
@@ -67,7 +67,7 @@ class APACHE_GEODE_EXPORT DistributedSystem {
    * Initializes the Native Client system to be able to connect to the
    * Geode Java servers. If the name string is empty, then the default
    * "NativeDS" is used as the name of distributed system.
-   * @throws IllegalStateException if GFCPP variable is not set and
+   * @throws IllegalStateException if GEODE_NATIVE variable is not set and
```

Comment:
To be consistent with the Java server, which uses GEODE_HOME, perhaps we should use GEODE_NATIVE_HOME for our environment variable.

Reply:
seems reasonable to me, I think I took [GEODE-4908](https://issues.apache.org/jira/browse/GEODE-4908) potentially to literally without checking what the java server does